### ISSUE TITLE
Develop connector modules

### ISSUE THREAD

**Issue by juanshishido** [hydrobase/analytics#3]
To "get" and "post" data to the database.

**Comment by binnie869** [comment]
I think we should use "pymongo" to connect and operate on the MongoDB

**Comment by juanshishido** [comment]
@binnie869 do you want to take the lead on this?

**Comment by binnie869** [comment]
@juanshishido : Yes, I am putting together something. Will send a PR when I am done!

**Comment by binnie869** [comment]
@hydrobase/core : what is the fictionality that we are expecting to expose using this module. I put down a few:
- Connect to DB
- Create a new collection
- Create a new user
- Add a document
- Add multiple documents
- Drop a collection
- Drop Documents
- Query a collection

What else?

**Comment by juanshishido** [comment]
Great question, @binnie869! Originally, I was only thinking of connecting to the DB, querying collections, and adding documents. As you point out, though, we may need additional functionality in the future as we develop new features. However, my suggestion is to start small (and with tests!).

In general, I would say it's best to work in small batches. A diff should not be more than one page long. This makes it much easier to review (and to write) code. But always start with tests.

Thanks!

**Comment by binnie869** [comment]
@juanshishido: Great comment. Start small and then we can build on that.  Will do so.

**Comment by brynamo** [comment]
@juanshishido + @binnie869 - can we clarify something...we are talking about on the data/analytics server correct? If we are talkinig about for the main application then I have a few things I'd like to weigh in on.

**Comment by juanshishido** [comment]
@brynamo Yes, this is just for the analytics side of things. This is for reading and writing to the database.

**Comment by binnie869** [comment]
@brynamo: I think the functions that are defined in this module would be very similar to one defined for the main application. 

Also, do we want a connection btw analytics server -> DB or we want to go via the node.js sever? Food for thought.

**Comment by brynamo** [comment]
I would say python server should connect directly. seems overly complicated with little benefit to go through Node. What do you @binnie869 is there a use case you are thinking of?
